springcloud-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/springcloud/DistributedCommandBusProperties.java

```diff
@@ -160,5 +166,21 @@ public String getFallbackUrl() {
         public void setFallbackUrl(String fallbackUrl) {
             this.fallbackUrl = fallbackUrl;
         }
+
+        /**
```

Comment:
Nit: I'd like to have body in the javadoc as well, even though it's a copy of what's stated on the `@return` line.